doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -174,74 +172,79 @@ maximize the impact of the mclock scheduler.
+7. After ensuring that the OSD nodes are in the desired configuration, run a
    simple 4KiB random write workload on the OSD(s) for 300 secs.
 8. Note the overall throughput(IOPS) obtained from the cbt output file. This
    value would be the baseline throughput(IOPS) with the default bluestore
    throttle options.
 9. If the intent is to determine the bluestore throttle values for your
-   environment, then set the two options, ``bluestore_throttle_bytes`` and
-   ``bluestore_throttle_deferred_bytes`` to 32 KiB(32768 Bytes) each to begin
-   with. Otherwise, you may skip to the next section.
+   environment, then set each of the two options ``bluestore_throttle_bytes``
+   and ``bluestore_throttle_deferred_bytes`` to 32 KiB(32768 Bytes) to start.
+   Otherwise, you may skip to the next section.
 10. Run the 4KiB random write workload as before on the OSD(s) for 300 secs.
 11. Note the overall throughput from the cbt log files and compare the value
-    against the baseline throughput in step 8.
+    against the baseline throughput acquired in step 8.
 12. If the throughput doesn't match with the baseline, increment the bluestore
     throttle options by 2x and repeat steps 9 through 11 until the obtained
     throughput is very close to the baseline value.
 
-For e.g., during benchmarking on a machine with NVMe SSDs a value of 256 KiB for
-both bluestore throttle and deferred bytes was determined to maximize the impact
-of mclock. For HDDs, the corresponding value was 40 MiB where the overall
-throughput was roughly equal to the baseline throughput. Note that in general
-for HDDs, the bluestore throttle values are expected to be higher when compared
-to SSDs.
+For example, during benchmarking on a machine with NVMe SSDs, a value of 256
+KiB for both "bluestore throttle" and "deferred bytes" was determined to
+maximize the impact of mclock. For HDDs, the corresponding value was 40 MiB,
+where the overall throughput was roughly equal to the baseline throughput. Note
+that in general, the bluestore throttle values for HDDs are expected to be
+higher than the bluestore throttle values for SSDs.
 
 .. _cbt: https://github.com/ceph/cbt
 
 
-Specify Max OSD Capacity
-------------------------
+Specifying Max OSD Capacity
+---------------------------
 
 The steps in this section may be performed only if the max osd capacity is
 different from the default values (SSDs: 21500 IOPS and HDDs: 315 IOPS). The
 option ``osd_mclock_max_capacity_iops_[hdd, ssd]`` may be set by specifying it
-in either the **[global]** section or in a specific OSD section **[osd.x]** of
-your Ceph configuration file.
+in either the **[global]** section or in a specific OSD section in your Ceph
+configuration file called **[osd.x]**.
 
 Alternatively, the following commands may be used,
 
-``$ ceph config set [global, osd] osd_mclock_max_capacity_iops_[hdd,ssd]
-<value>``
+.. prompt:: bash $
+
+  ceph config set [global, osd] osd_mclock_max_capacity_iops_[hdd,ssd]
+  <value>
```

Comment:
<value> should be a part of the previous line:

ceph config set [global, osd] osd_mclock_max_capacity_iops_[hdd,ssd] <value>

Reply:
Done, but I'm not sure if it makes a difference when the docs are rendered.